spyder_line_profiler/spyder/widgets.py

```diff
@@ -68,7 +58,29 @@
 
 WEBSITE_URL = 'http://pythonhosted.org/line_profiler/'
 
+locale_codec = QTextCodec.codecForLocale()
+
+
+def is_lineprofiler_installed():
+    """
+    Checks if the program and the library for line_profiler is installed.
+    """
+    return (programs.is_module_installed('line_profiler')
+            and programs.find_program('kernprof') is not None)
+
+class PatchPythonModulesComboBox(PythonModulesComboBox):
+    # for backwards compatibility to Spyder<5.1.5, where no id_ is used
+    def __init__(self, parent, adjust_to_contents=False, id_=None, **kwargs):
+        try:
+            PythonModulesComboBox.__init__(self, parent=parent, id_=id_,
+                                   adjust_to_contents=adjust_to_contents)
+        except Exception:
+            PythonModulesComboBox.__init__(self, parent=parent,
+                                       adjust_to_contents=adjust_to_contents)
```

Comment:
I think that with the changes in the CI workflows configs this shouldn't be needed but lets check first applying the config🤞🏼

Reply:
Ok, I removed the patch